packages/server/src/course/course.controller.ts

```diff
@@ -556,6 +557,34 @@ export class CourseController {
     }
   }
 
+  @Get(':id/get_user_info/:page/:role?')
+  @UseGuards(JwtAuthGuard, CourseRolesGuard)
+  @Roles(Role.PROFESSOR)
+  async getUserInfo(
+    @Param('id') courseId: number,
+    @Param('page') page: number,
+    @Param('role') role?: Role,
+    @Query('search') search?: string,
+  ): Promise<UserPartial[]> {
+    const pageSize = 50;
+    try {
```

Comment:
probably don't need this second try catch here if you have it at the service level

Reply:
true, good catch!

Reply:
o wait ima remove both bc i don't think it can error out